src/SmartComponents/BaselinesTable/redux/helpers.js

```diff
@@ -3,9 +3,34 @@ function buildBaselinesTable(data, selectedBaselineIds) {
 
     data.forEach(function(baseline) {
         let row = [];
+        let monthMap = [ 'Jan', 'Feb', 'Mar', 'Apr', 'May', 'June', 'July', 'Aug', 'Sept', 'Oct', 'Nov', 'Dec' ];
```

Comment:
Is this the same format as used elsewhere in the apps? Some have 3 char, but others have 4